tests/integration/standard/test_authentication_misconfiguration.py

```diff
@@ -20,6 +20,12 @@
 
 class MisconfiguredAuthenticationTests(unittest.TestCase):
     """ One node (not the contact point) has password auth. The rest of the nodes have no auth """
+    # TODO: 	Fix ccm to apply following options to scylla.yaml
```

Comment:
what is that comment ?

Reply:
This test fails due to the ccm issue, this comment to fix it in future.